### ISSUE TITLE
Useful Query in Queries.md is incorrect

### ISSUE THREAD

**Issue by iKuba** [MetricsGrimoire/CVSAnalY#44]
What is the intention of the queries found in section 44?

The explanation as follows: Project size in LOC / SLOC per unit of time leads me to believe that the goal of the query is to finding the TOTAL LOC/SLOC of the project over time. The provided query only produces the LOC/SLOC of the files committed in a given period and does not take into account that files may be created and deleted in that period which I do not think is correct.

**Comment by andygrunwald** [comment]
Hey @iKuba,

you are right. So you propose to measure the _touched_ lines over time?
THis might make sense. Thanks for this.
Did you got a solution for this task? An alternative query? Or maybe a better description for the current query?

Sadly at the moment i got no time to have a deeper look at it. But maybe you got an good idea.

**Comment by iKuba** [comment]
I do have a solution, unfortunately it's somewhat messy. I have exams this week so I will get back to you after they are over.

**Comment by andygrunwald** [comment]
It does not matter that is messy. We can clean up :)
Thanks! And good luck for your exams.

**Comment by andygrunwald** [comment]
Hey @iKuba, 
do you passed your exams? I hope so.
Can you provide the mentioned solution? Thanks!

**Comment by iKuba** [comment]
Yes, I did thanks! I will have to check if I still have this query.

**Comment by andygrunwald** [comment]
@iKuba Any updates here?

**Comment by iKuba** [comment]
I don't have access to the source anymore ):

It wasn't a very nice query if I remember correctly..  It involved some
really complicated unions so it probably wasn't the best solution anyways...
